runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java

```diff
@@ -63,11 +63,11 @@ default boolean swapAssignment()
 	}
 
 	@ConfigItem(
-		keyName = "swapBanker",
-		name = "Bank",
-		description = "Swap Talk-to with Bank on Bank NPC<br>Example: Banker"
+		keyName = "swapBankExchange",

```

Comment:
changing keynames for no reason is probably bad, it would change peoples current config options